Example/Kommunicate_ExampleUITests/KommunicateCreateConversationAndSendMessagesTests.swift

```diff
@@ -19,6 +19,7 @@ class KommunicateCreateConversationAndSendMessagesTests: XCTestCase {
     override func setUp() {
         super.setUp()
         continueAfterFailure = false
+        sleep(15)
```

Comment:
See if we can reduce the sleep time, otherwise it will increase build time.

Reply:
@mukeshthawani  done reduced to 10 sec